client/my-sites/google-my-business/new-account/index.js

```diff
@@ -0,0 +1,100 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class GoogleMyBusinessNewAccount extends Component {
+	static propTypes = {
+		recordTracksEvent: PropTypes.func.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	goBack = () => {
+		page.back( `/google-my-business/${ this.props.siteSlug }/select-business-type` );
+	};
+
+	trackCreateMyListingClick = () => {
+		this.props.recordTracksEvent(
+			'calypso_google_my_business_new_account_create_my_listing_button_click'
+		);
+	};
+
+	trackNoThanksClick = () => {
+		this.props.recordTracksEvent(
+			'calypso_google_my_business_new_account_no_thanks_button_click'
+		);
+	};
+
+	render() {
+		const { siteSlug, translate } = this.props;
+
+		return (
+			<Main className="gmb-new-account" wideLayout>
+				<PageViewTracker
+					path="/google-my-business/:site/new"
+					title="Google My Business > New"
+				/>
+
+				<HeaderCake isCompact={ false } alwaysShowActionText={ false } onClick={ this.goBack }>
+					{ translate( 'Google My Business' ) }
+				</HeaderCake>
+
+				<Card className="gmb-new-account__card">
+					<img
+						alt={ translate( 'Local business illustration' ) }
+						className="gmb-new-account__illustration"
+						src="/calypso/images/google-my-business/business-local.svg"
+					/>
+
+					<h1 className="gmb-new-account__heading">
+						{ translate( 'It looks like you might be new to Google My Business' ) }
+					</h1>
+
+					<p>
+						{ translate(
+							'Google My Business lists your local business on Google Search and Google Maps. ' +
+							'It works for businesses that have a physical location or serve a local area'
+						) }
+					</p>
+
+					<div className="gmb-new-account__actions">
+						<Button primary onClick={ this.trackCreateMyListingClick }>
+							{ translate( 'Create My Listing' ) }
+						</Button>
+
+						<Button href={ `/stats/${ siteSlug }` } onClick={ this.trackNoThanksClick }>
```

Comment:
Why here we don't use `goBack()`?

Reply:
I think the reasoning is that it doesn't make sense for the user to go back to the `Select Business Type` page since the outcome will be the same.